edxval/api.py

```diff
@@ -247,7 +247,7 @@ def get_video_transcript_data(video_ids, language_code):
         try:
             video_transcript = VideoTranscript.objects.get(video_id=video_id, language_code=language_code)
             transcript_data = dict(
-                file_name=video_transcript.transcript.name,
+                file_name=video_transcript.filename,
```

Comment:
can this effect existing usages of this `get_video_transcript_data` function?

Reply:
No it will not.

Reply:
sandbox is up with these changes, you may test there.